modules/docs/src/main/tut/docs/12-Custom-Mappings.md

```diff
@@ -62,8 +62,8 @@ HPS.set(1, ("foo", 42))
 // Or leave the 1 out if you like, since we usually start there
 HPS.set(("foo", 42))
 
-// Which simply delegates to the Composite instance
-Composite[(String,Int)].set(1, ("foo", 42))
+// Which simply delegates to the Write instance
+Write[(String,Int)].set(1, ("foo", 42))
 ```
 
 **doobie** can derive `Composite` instances for primitive column types and options thereof, plus tuples, `HList`s, shapeless records, and case classes whose elements have `Composite` instances. These primitive column types are identified by `Meta` instances, which describe single-column mappings.
```

Comment:
It looks like there are some leftovers of `Composite` in the docs

Reply:
Ah yeah I need to update the doc. Thanks.